test/old_tests/UnitTests/apartment_context.cpp

```diff
@@ -46,6 +46,11 @@ namespace
         co_await context;
     }
 
+// Not yet buildable on mingw-w64. The lambda needs to have __stdcall
```

Comment:
Since this lambda has no captures, it can be converted to a normal function with __stdcall.